### DISCUSSION TITLE
Release navigattion

### DISCUSSION THREAD

**Discussion by Heshmatkhah** [community/community#23888] [New to GitHub]
<p>Hi<br>
I want to navigate releases of <a href="https://github.com/brave/brave-browser" rel="noopener nofollow ugc">brave’s git repo</a> to find a specific old version based on chromium version 79, which supports the adobe flash.<br>
I <strong>HAVE TO</strong> use an old website that is using flash.<br>
Navigating releases in the current UI is painful.<br>
I can’t skip multiple pages; changing url parameters makes next/previous buttons buggy.<br>
Is there a better way to seek releases?</p>

**Comment by jsoref** [comment, marked as answer]
<p>Yes. Check the changelog.</p><aside class="onebox githubblob" data-onebox-src="https://github.com/brave/brave-browser/blob/0ddaf339197d875281796d3d21f7a5cff3f532e2/CHANGELOG_DESKTOP.md">
  <header class="source">

      <a href="https://github.com/brave/brave-browser/blob/0ddaf339197d875281796d3d21f7a5cff3f532e2/CHANGELOG_DESKTOP.md" target="_blank" rel="noopener">github.com</a>
  </header>

  <article class="onebox-body">
    <h4><a href="https://github.com/brave/brave-browser/blob/0ddaf339197d875281796d3d21f7a5cff3f532e2/CHANGELOG_DESKTOP.md" target="_blank" rel="noopener">brave/brave-browser/blob/0ddaf339197d875281796d3d21f7a5cff3f532e2/CHANGELOG_DESKTOP.md</a></h4>


    <pre><code class="lang-md"># Changelog

## [1.29.77](https://github.com/brave/brave-browser/releases/tag/v1.29.77)

 - Fixed not being able to download files from Google Drive due to cookies being blocked. ([#17795](https://github.com/brave/brave-browser/issues/17795))
 - Upgraded Chromium to 93.0.4577.63. ([#17790](https://github.com/brave/brave-browser/issues/17790)) ([Changelog for 93.0.4577.63](https://chromium.googlesource.com/chromium/src/+log/93.0.4577.58..93.0.4577.63?pretty=fuller&amp;n=1000))

## [1.29.76](https://github.com/brave/brave-browser/releases/tag/v1.29.76)

 - Added Gemini two-way Brave Rewards wallet. ([#17161](https://github.com/brave/brave-browser/issues/17161))
 - Added prompt to enable crash reporting. ([#1841](https://github.com/brave/brave-browser/issues/1841))
 - Added support for launching Brave with a Tor window when using the terminal/command line interface. ([#2105](https://github.com/brave/brave-browser/issues/2105))
 - Added "Show Search Tabs button" setting under brave://settings/appearance. ([#16007](https://github.com/brave/brave-browser/issues/16007))
 - Added "walletActive" parameter to the stats usage ping. ([#17059](https://github.com/brave/brave-browser/issues/17059))
 - Added ephemeral support to "BroadcastChannel". ([#17220](https://github.com/brave/brave-browser/issues/17220))
 - Added Uphold user ID under brave://rewards-internals to assist with rewards support. ([#16937](https://github.com/brave/brave-browser/issues/16937))
 - Added a "Relaunch" button which displays after the in-line "Tips Settings" are modified. ([#16941](https://github.com/brave/brave-browser/issues/16941))
 - Changed the location of QR code for iOS devices to be under the "Manage Your Wallet" modal on brave://rewards page. ([#16954](https://github.com/brave/brave-browser/issues/16954))
 - Updated the date of installation parameter in the stats ping to expire in 30 days. ([#17089](https://github.com/brave/brave-browser/issues/17089))
 - Reduced the height of the download bar. ([#1638](https://github.com/brave/brave-browser/issues/1638))
</code></pre>


  This file has been truncated. <a href="https://github.com/brave/brave-browser/blob/0ddaf339197d875281796d3d21f7a5cff3f532e2/CHANGELOG_DESKTOP.md" target="_blank" rel="noopener">show original</a>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
